src/hooks/useUser.js

```diff
@@ -1,10 +1,12 @@
 import { useQuery } from '@apollo/client';
 import { CURRENT_USER_QUERY } from 'queries'
 
-export const useUser = () => {
+function useUser() {
```

Comment:
just something to think about, could maybe do something like:

```javascript 
const { data, loading } = useQuery(CURRENT_USER_QUERY);

if (!loading) {
const {currentUser} = data;
return currentUser;
}
```

also feel like useUser is a little weird tongue twister of a name lol, maybe we can do useCurrentUser?

Reply:
Good thinking! Changed the name and updated all imports. Refactored as suggested here adding in a possible error state as well:

```
if (!loading && !error)
```